### DISCUSSION TITLE
Welcome to activerecord-jdbc-adapter Discussions!

### DISCUSSION THREAD

**Discussion by headius** [jruby/activerecord-jdbc-adapter#1147] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesseChavez** [comment]
Hi @headius

Do you guys need help with updating the adapter to support Rails 7.1 and 7.2 ?

I've got some time allocated at work to update our fork [activerecord-jdbc-alt-adapter](https://github.com/JesseChavez/activerecord-jdbc-adapter) to support mssql and postgres with the latest rails.
